src/auth-service/routes/v2/users.js

```diff
@@ -403,7 +394,7 @@ router.get(
 );
 
 router.get(
-  "/verify/:user_id/:token/:category",
```

Comment:
@FaithDaka , just removed the "category" path which  was recently introduced for reasons I do not clearly remember. This is what made the endpoint "not available" as you earlier shared via Slack DM. cc: @BenjaminSsempala